config/routing.yml

```diff
@@ -1,3 +1,9 @@
+# Dynamic routing for Pages 2.0.x
+phpbb_page_dynamic_controller:
```

Comment:
I don't see your point. It's already very generic (the loader is less than 10 loc which are only related to the Pages extension logic).

If it can looks weird here it's just because we are in an edge case: a loader which can be used only once per request and without any argument.

About the type, it's just a name and up to @VSEphpbb to choose it.